deploy/olm-catalog/splunk/1.0.3/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     name: v2
     schema:
       openAPIV3Schema:
-        description: ClusterMaster is the Schema for the clustermasters API
+        description: ClusterMaster is the Schema for the clustermanagers API
```

Comment:
Hmm, don't think the olm files under 1.0.3 need to be included, as that release is already completed

Reply:
Oh okay. Make sense. Let me do that.